src/main/java/uk/gov/hmcts/reform/cpo/payload/UpdateCasePaymentOrderRequest.java

```diff
@@ -30,13 +34,9 @@
 
     @NotEmpty(message = ValidationError.CASE_ID_REQUIRED)
     @ValidCaseId
-    @ApiModelProperty(value = "Case Id for which the record applies", required = true, example = "1617288400189308")
+    @ApiModelProperty(value = "Case Id for which the record applies", required = true, example = "2061729969689088")
     private String caseId;
 
-    @NotEmpty(message = ValidationError.ORDER_REFERENCE_REQUIRED)
-    @ApiModelProperty(value = "Description of the Payments system order reference", required = true, example = "1111")
-    private String orderReference;
-
     @NotEmpty(message = ValidationError.ACTION_REQUIRED)
     @ApiModelProperty(value = "Action that initiated the creation of the case payment order", required = true,
```

Comment:
very minor comment, value could be constant as its used here and in CreateCasePaymentOrderRequest.java

Reply:
Sonar has pointed out the same duplication so I have resolved in c80421b353528a35e29094b40e05c56608035b19